src/argus_htmx/incidents/views.py

```diff
@@ -23,8 +23,9 @@
 def prefetch_incident_daughters():
     return (
         Incident.objects
-        .select_related("source")
+        .defer("metadata", "search_text")
         .prefetch_related(
```

Comment:
I'm not in favor of deferring metadata loading in the incidents listing since we will need the metadata there. I'm concerned that this might lead to n+1 queries since now we will read/query metadata in the template